### ISSUE TITLE
[yggtorrent] Exception (yggtorrent): Login failed: Selector

### ISSUE THREAD

**Issue by InTheFlipside** [Jackett/Jackett#10910]
Hi, I got this error when using the YGGTorrent Website

` An error occurred while testing this indexer
Exception (yggtorrent): Login failed: Selector "div#top_panel:contains("Déconnexion")" didn't match: Parse error`

https://i.imgur.com/DbJHFtz.png

**Comment by LacazeThomas** [comment]
I don't think so, i tried with v0.17.348 but same problem

**Comment by ilike2burnthing** [comment]
The site has been having issues of late. A few changes have been made to the indexer as a result of this, and to reduce the number of requests we make to the site when logging in. I can't see anything further which would be necessary.

As long as Jackett is up to date, the site link is `https://www2.yggtorrent.si/`, and your details are correct, I can only point at the site being the issue.

---

TL;DR
>seems to be random
